backend/src/layer/data/swka_parser/html_parser.rs

```diff
@@ -112,23 +115,28 @@ lazy_static! {
     static ref ENV_SCORE_NODE_CLASS_SELECTOR: Selector = Selector::parse("div.enviroment_score.average").expect(SELECTOR_PARSE_E_MSG);
 
     /// A Regex for getting prices in euros. A price consists of 1 or more digits, followed by a comma and then exactly two digits.
-    static ref PRICE_REGEX: Regex = Regex::new(r"([0-9]*),([0-9]{2})").expect(REGEX_PARSE_E_MSG);
+    static ref DECIMAL_REGEX: Regex = Regex::new(r"([0-9]*),([0-9]{2})").expect(REGEX_PARSE_E_MSG);
```

Comment:
I think this rename can lead to dangerous futures: is it clear that a "decimal" always has exactly two decimal digits? This is an essential requirement for the price parsing below in rust code.
I think this rename loses that semantic, maybe a decimal does not even need to have exactly two decimal places